data_structures/binary_tree/AVLtree.py

```diff
@@ -236,21 +234,4 @@ def traversale(self): #a level traversale, gives a more intuitive look on the tr
         print()
         print("*************************************")
         return
-    
-    def test(self):
-        getheight(None)
-        print("****")
-        self.getheight()
-if __name__ == "__main__":
```

Comment:
This main should be left here, since it prints something (in the `traversale` method).